src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -222,6 +222,8 @@ public class SlaveTemplate implements Describable<SlaveTemplate> {
 
     public int maxTotalUses;
 
+    private boolean avoidUsingOrphanedNodes;
```

Comment:
As a rule, we prefer to avoid negative terminology in options. Would have been clearer (IMO) to name the option, e.g., `reuseOrphanNodes`, then have it default to `true` in `readResolve` (using `Boolean`) for compatibility reasons (probably leave it off for newly configured clouds).